PyRoute/Calculation/TradeCalculation.py

```diff
@@ -148,6 +148,11 @@ def calculate_routes(self):
         btn = [(s, n, d) for (s, n, d) in self.galaxy.ranges.edges(data=True) if s.component == n.component]
         btn.sort(key=lambda tn: tn[2]['btn'], reverse=True)
 
+        # Pick landmark - biggest WTN system in the biggest graph component
+        stars = [item for item in self.galaxy.stars if item.component == self.galaxy.big_component]
+        stars.sort(key=lambda item: item.wtn, reverse=True)
+        stars[0].is_landmark = True
```

Comment:
This sets one star to be a landmark (to start). Did you want to set all of the largest (e.g. WTN 13) systems to be landmarks?

Reply:
I'm still going back and forth on this.  I don't want too many landmarks because of the performance hit in heuristic calculation - assuming we're using landmarks for bounds, each check would have to spin through each landmark in the same component as the stars being checked.

To grab multiple (say 3) landmarks, I'd want to do it iteratively.

Start with the biggest WTN in the component.
The next landmark in that component has WTN of at least 1 less than the big boy (WTN 12+ in this case), and is furthest away in the component (so the geometry actually helps the bound).
Likewise, landmark 3 would be the furthest away in the component from the other two.
Mosh, rinse, repeat as needed.

Reply:
@tjoneslo , this has been superseded by the approximate-shortest-path-tree work